### PR TITLE
Barriers fix

### PR DESCRIPTION
**Pull request by babenek**
## Description

Please include a summary of the change and which is fixed.

- Fix Bariers issue in regex - second value of the same rule was not found
- update scores of benchmark after https://github.com/Samsung/CredData/pull/80
- refactored test with data reports to update them automaticaly
- removed extra requirements.txt file - one Dependabot issue for onnx
- update readme for experiment activity
- fixes: https://github.com/Samsung/CredSweeper/security/dependabot/11 

## How has this been tested?

Please describe the tests that you ran to verify your changes.

- [x] UnitTest
- [x] Benchmark

### CODE REVIEW

File: .github/workflows/benchmark.yml [hunk 1]
```diff
@@ -336,7 +336,7 @@ jobs:
       - name: Checkout CredData
         uses: actions/checkout@v3
         with:
-          repository: Samsung/CredData
+          repository: babenek/CredData
```
Comment (babenek) [inline .github/workflows/benchmark.yml:339]:
```suggestion
          repository: Samsung/CredData
```

File: .github/workflows/benchmark.yml [hunk 2]
```diff
@@ -163,7 +163,7 @@ jobs:
       - name: Checkout CredData
         uses: actions/checkout@v4
         with:
-          repository: Samsung/CredData
+          repository: babenek/CredData
```
Comment (babenek) [inline .github/workflows/benchmark.yml:166]:
```suggestion
          repository: Samsung/CredData
```

File: .github/workflows/benchmark.yml [hunk 3]
```diff
@@ -62,7 +62,7 @@ jobs:
...
```
Comment (babenek) [inline .github/workflows/benchmark.yml:65]:
```suggestion
          repository: Samsung/CredData
```

File: .github/workflows/benchmark.yml [hunk 4]
```diff
@@ -22,7 +22,7 @@ jobs:
...
```
Comment (babenek) [inline .github/workflows/benchmark.yml:25]:
```suggestion
          repository: Samsung/CredData
```

**Review by babenek — COMMENTED** [review]
TODO: rollback after https://github.com/Samsung/CredData/pull/81

**Review by xDizzix — APPROVED** [review]

**Review by iuriimet — APPROVED** [review]

**Pull request merged**